assets/js/base/components/payment-methods/express-payment-methods.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import {
+	useCheckoutData,
+	usePaymentEvents,
+	useExpressPaymentMethods,
+} from '@woocommerce/base-hooks';
+import { cloneElement } from '@wordpress/element';
+
+const ExpressPaymentMethods = () => {
+	const [ checkoutData ] = useCheckoutData();
+	const { dispatch, select } = usePaymentEvents();
+	// not implementing isInitialized here because it's utilized further
+	// up in the tree for express payment methods. We won't even get here if
+	// there's no payment methods after initialization.
+	const { paymentMethods } = useExpressPaymentMethods();
+	const paymentMethodSlugs = Object.keys( paymentMethods );
+	const content =
+		paymentMethodSlugs.length > 0 ? (
+			paymentMethodSlugs.map( ( slug ) => {
+				const expressPaymentMethod =
+					paymentMethods[ slug ].activeContent;
+				const paymentEvents = { dispatch, select };
+				return (
+					<li
+						key={ `paymentMethod_${ slug }` }
```

Comment:
I did it mostly to prevent potential collision with `slug` getting used as a key in other contexts.  But I _just_ learned that it only needs to be unique among it's siblings, so ya this can be reduced to just be the `slug`.  Good catch!

Reply:
done in 6853d21